Examples/DemoARViewController.swift

```diff
@@ -380,6 +380,17 @@ final class DemoARViewController: UIViewController, ARSCNViewDelegate, ARSession
     private var session: ARSession {
         return arView!.session
     }
+    
+    // MARK: - dummy data, should be handled in GPX
```

Comment:
So why isn't it being handled there? This comment makes it feel like it's not supposed to be here